src/oc_erchef/apps/data_collector/rebar.config

```diff
@@ -15,9 +15,10 @@
 ]}.
 
 {deps, [
-    {lager, ".*", {git, "https://github.com/basho/lager", {tag, "2.1.1"}}},
-    {opscoderl_httpc, ".*", {git, "https://github.com/chef/opscoderl_httpc", {branch, "master"}}},
-    {pooler, ".*", {git,"git://github.com/seth/pooler.git", {branch,"master"}}}
+    {lager, ".*", {git, "https://github.com/basho/lager.git", {tag, "2.1.1"}}},
+    {opscoderl_httpc, ".*",
+     {git, "https://github.com/chef/opscoderl_httpc.git", {branch, "ma/ibrowse_uses_mainline"}}},
```

Comment:
any reason to use ma/ibrowse_uses_mainline? looks like we have merged it in the opscoderl_httpc repo.

Reply:
Fixed.